sklearn/tree/_tree.pyx

```diff
@@ -551,14 +728,17 @@ cdef class Gini(ClassificationCriterion):
 
 
 cdef class RegressionCriterion(Criterion):
-    """Abstract criterion for regression.
+    """A regression criteria. 
```

Comment:
Why did your remove "Abstract"? This is important information: it cannot be used directly as it does not implement. It's just there to factorize common cinit and dealloc for the `MSE` and `FriedmanMSE` concrete classes.